src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.android.cs

```diff
@@ -0,0 +1,557 @@
+using Android.Content;
+using Android.Content.Res;
+using Android.Graphics.Drawables;
+using Android.OS;
+using Android.Views;
+using Android.Views.Accessibility;
+using Android.Widget;
+using Microsoft.Maui.Controls.Compatibility.Platform.Android;
+using static System.OperatingSystem;
+using AView = Android.Views.View;
+using Color = Android.Graphics.Color;
+using MColor = Microsoft.Maui.Graphics.Color;
+using MView = Microsoft.Maui.Controls.View;
+using PlatformView = Android.Views.View;
+using ParentView = Android.Views.IViewParent;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+public partial class TouchBehavior
+{
+	private static readonly MColor defaultNativeAnimationColor = MColor.FromRgba(128, 128, 128, 64);
+	private bool isHoverSupported;
+	private RippleDrawable? ripple;
+	private AView? rippleView;
+	private float startX;
+	private float startY;
+	private MColor? rippleColor;
+	private int rippleRadius = -1;
+	private AView? view = null;
+	private ViewGroup? viewGroup;
+
+	private AccessibilityManager? accessibilityManager;
+	private AccessibilityListener? accessibilityListener;
+
+	private bool IsAccessibilityMode => accessibilityManager is not null
+	                                    && accessibilityManager.IsEnabled
+	                                    && accessibilityManager.IsTouchExplorationEnabled;
+
+	private readonly bool isAtLeastM = IsAndroidVersionAtLeast((int) BuildVersionCodes.M);
```

Comment:
this value will be the same for all instances, and we with that we call android APIs just once.
```suggestion
	static readonly bool isAtLeastM = IsAndroidVersionAtLeast((int) BuildVersionCodes.M);
```

Reply:
Done